### PR TITLE
[docs] update README for ping command

### PR DESCRIPTION
**Pull request by superwhd**
- Add `-I interface` argument for ping command.
- Update ping examples

### CODE REVIEW

File: src/cli/README.md [hunk 1]
```diff
@@ -1942,21 +1942,27 @@ Set the preferred Thread Leader Partition ID.
 Done
 ```
 
-### ping \<ipaddr\> \[size\] \[count\] \[interval\] \[hoplimit\] \[timeout\]
+### ping \[-I interface\] \<ipaddr\> \[size\] \[count\] \[interval\] \[hoplimit\] \[timeout\]
```
Comment (librasungirl) [inline src/cli/README.md:1945]: update to be more intuitive?
```suggestion
### ping \[-I source\] \<ipaddr\> \[size\] \[count\] \[interval\] \[hoplimit\] \[timeout\]
```

File: src/cli/README.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ Done
 - [parent](#parent)
 - [parentpriority](#parentpriority)
 - [partitionid](#partitionid)
-- [ping](#ping-ipaddr-sizecount-intervalhoplimit)
+- [ping](#ping--i-interface-ipaddr-size-count-interval-hoplimit-timeout)
```
Comment (librasungirl) [inline src/cli/README.md:84]: update for consistency?

File: src/cli/README.md [hunk 3]
```diff
@@ -1942,21 +1942,27 @@ Set the preferred Thread Leader Partition ID.
 Done
 ```
 
-### ping \<ipaddr\> \[size\] \[count\] \[interval\] \[hoplimit\] \[timeout\]
+### ping \[-I source\] \<ipaddr\> \[size\] \[count\] \[interval\] \[hoplimit\] \[timeout\]
 
 Send an ICMPv6 Echo Request.
 
+- interface: The source IPv6 address of the echo request.
```
Comment (librasungirl) [inline src/cli/README.md:1949]: update for consistency?

**Review by Irving-cl — APPROVED** [review]
LGTM :+1:

**Review by librasungirl — APPROVED** [review]

**Review by jwhui — APPROVED** [review]

**Pull request merged**